modules/card/react/stories/stories.mdx

```diff
@@ -0,0 +1,27 @@
+import {Meta, Story, Preview, Props} from '@storybook/addon-docs/blocks';
+
+import {spacing} from '../../../core/react/index';
+import README from '../README.md';
+import Card from '../index';
+
+<Meta title="Card" component={Card} />
+
+# Card
+
+<Preview>
+  <Story name="Default">
+    <div className="story">
+      <Card heading="Title">Card contents</Card>
+    </div>
+  </Story>
+</Preview>
+
+<Props of={Card} />
+
+## No Padding
+
+<Preview>
```

Comment:
Yeah, this was an example...you'll see that without the story tab these will be excluded from the navigation.

Reply:
I've been thinking about this. @sahlhoff and I talked about the possibility of a story without showing up in the documentation for ChromaticQA or Cypress. I think if we stick with Storybook as the documentation frontend, we can just call these out as sections. Our Accessibility team runs audits on our stories. I think it would be good to call out sections. Like this section shows all the valid states of a component for manual and automated verification and this section shows a contextual example with approved accessibility features. Not always can we control accessibility, but our examples should show an implementation that matches a11y specs so application developers have a working example they can follow.